### ISSUE TITLE
Color contrast false positive

### ISSUE THREAD

**Issue by testingtycoon** [dequelabs/axe-core#608]
I have analyzed below page and observed axe throws color contrast issues that are not actually issues when tested them manually using color contrast analyzer. I didn't understand how axe picks foreground color. Here are the page details.

Page URL: https://www.udemy.com/corejava/learn/v4/content
Credentials: mailed to Wilco
![colorcontrast_falsepositive](https://user-images.githubusercontent.com/33425105/32497897-14c2d480-c3f4-11e7-90fa-fba4760aa5eb.PNG)

Elements to focus: Time duration values (ex: 17:10) under course content tab. Please find the attached screenshot.

Axe identified foreground and back ground colors:  #959aa2 and #ffffff
Manually identified foreground and back ground colors: #686f7a and #ffffff
axe-core version: 2.4.1

**Comment by marcysutton** [comment]
Without logging in, I can tell you a bit about how to understand the results. The foreground element is the target under `Element source`, and the background is the `Related node` on the right side. If the element in question isn't the one supplying a background color, then you've found a bug. But I can't know for sure until I can look at the DOM tree.

**Comment by erikkroes** [comment]
I've found a page on udemy with (I assume) the same markup.

![screen shot 2017-11-13 at 15 36 40](https://user-images.githubusercontent.com/10405024/32730860-a5f4f288-c888-11e7-8606-44775d0d0e68.png)

It looks like the span has its opacity changed to 0.7 through css. I created a file in photoshop with a white background, and a layer of #686f7a set to 70% opacity to double-check. I get the exact same value aXe gives: #959aa2
